api/src/pcapi/sandboxes/scripts/creators/test_cases/__init__.py

```diff
@@ -82,3 +177,58 @@ def create_offers_with_gtl_id(gtl_id: str, size_per_gtl: int, venue: offerers_mo
             size=random.randint(1, 10),
             offer=offer,
         )
+
+
+def create_offers_with_same_ean() -> None:
+    offers = []
+    product = offers_factories.ProductFactory(
+        name="Le livre du pass Culture",
+        subcategoryId=subcategories_v2.LIVRE_PAPIER.id,
+        extraData={
+            "ean": "9782070360024",
+            "author": "Ramon Hahn",
+        },
+    )
+    for venue_data in venues:
+        offers.append(
+            offers_factories.OfferFactory(
+                product=product,
+                isActive=True,
+                venue=offerers_factories.VenueFactory(
+                    name=venue_data["name"],
+                    venueTypeCode=offerers_models.VenueTypeCode.BOOKSTORE,
+                    isPermanent=True,
+                    latitude=venue_data["latitude"],
+                    longitude=venue_data["longitude"],
+                    address=venue_data["address"],
+                    postalCode=venue_data["postalCode"],
+                    city=venue_data["city"],
+                    departementCode=venue_data["departementCode"],
+                ),
+                extraData={
+                    "ean": "9782070360024",
+                    "author": "Ramon Hahn",
+                },
+            )
+        )
+        for offer in offers:
+            offers_factories.StockFactory.create_batch(
+                size=random.randint(1, 10),
+                offer=offer,
+            )
+    for _ in range(10):
+        ean = "978" + str(random.randint(1000000000, 9999999999))
+        product = offers_factories.ProductFactory(
+            subcategoryId=subcategories_v2.LIVRE_PAPIER.id,
+            extraData={"ean": ean, "author": factory.Faker("name")},
+        )
+        offer = offers_factories.OfferFactory(
+            isActive=True,
+            product=product,
+            extraData={"ean": ean, "author": factory.Faker("name")},
+            venue=offers[0].venue,
+        )
+        offers_factories.StockFactory.create_batch(
+            size=random.randint(1, 10),
+            offer=offer,
+        )
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏C'est bizarre d'avoir ça dans `create_offers_with_same_ean`
j'aurais extrait dans sa propre fonction